apps/frontend/src/components/Modal.tsx

```
@@ -7,14 +7,32 @@ interface ModalProps {
 
-export const Modal = ({ title, content, onClose, maxWidth = 500 }: ModalProps) => {
+export const Modal = ({ title, content, onClose, maxWidth = 500, padding = true }: ModalProps) => {
   const theme = useTheme();
 
   return (
-    <div css={modalOverlayStyle} onClick={onClose}>
-      <div css={modalContentStyle(theme, maxWidth)} onClick={e => e.stopPropagation()}>
+    <div
+      css={modalOverlayStyle}
+      onClick={onClose}
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)